src/main/java/com/asu/cse545/group12/domain/Users.java

```diff
@@ -58,12 +58,12 @@ public String toString() {
 	@Column(name = "GENDER")
 	private String gender;
 	
-	@NotNull
+	//@NotNull
     //@NotEmpty//(message = "Please enter your Username.")

```

Comment:
We are going to handle it in validator. Currently we are using same form
for creating external user and internal user and system administrator
doesn't input username and password while creating internal user. So I
comment it
On Oct 16, 2015 10:22 PM, "Nagarjuna Myla" <notifications@github.com> wrote:

> In src/main/java/com/asu/cse545/group12/domain/Users.java
> <https://github.com/chasethenag420/sbs/pull/50#discussion_r42306111>:
>
> > @@ -58,12 +58,12 @@ public String toString() {
> >  	@Column(name = "GENDER")
> >  	private String gender;
> >  	
> > -	@NotNull
> > +	//@NotNull
> >      //@NotEmpty//(message = "Please enter your Username.")
>
> Shouldn't we uncomment @Notempty <https://github.com/Notempty> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chasethenag420/sbs/pull/50/files#r42306111>.
>